app/views/users/index.html.erb

```diff
@@ -1 +1,3 @@
-<p>Here is a list of users</p>
\ No newline at end of file
+<div class="user-list">
+  <%= render partial: "shared/user_card", collection: @users.order(id: :asc), as: :user, spacer_template: "shared/spacer" %>
```

Comment:
:heavy_check_mark: Great job using a `partial` with a `collection` parameter to render your collections.

Reply:
Thank you.